### PR TITLE
chore: update content on solutions/for-surveyors

### PR DESCRIPTION
**Pull request by DenisePeterson**

### CODE REVIEW

File: src/pages/solutions/for-surveyors.astro [hunk 1]
```diff
@@ -9,50 +10,88 @@ import { Image } from 'astro:assets';
...
     <Image slot="image" src={agreementPhoto} alt="two people shaking hands in agreement" class="w-full max-w-[400px]" />
-    <p>text</p>
+    <p>
+      Surveyors have long relied on GPS for their land surveying needs. When GNSS networks came along, offering more
+      reliable, high-precision accuracy, the survey industry was one of the earliest adopters of the new technology.
+    </p>
+    <p>
+      Today, networks such as ours offer coverage across the globe. Our TURN and Reno networks provides coverage in not
+      only Utah, but Nevada, Wyoming, and Idaho. We continue to offer surveryors a reliable network and expert support.
+    </p>
```
Comment (steveoh) [inline src/pages/solutions/for-surveyors.astro:49]: **issue**: this is supposed to be providing solutions to problems for surveyors. This isn't achieving that. We want to market to survyors here. Why do they want to use the network. What does it give them. This paragraph should probably be replaced.

File: src/pages/solutions.astro [hunk 2]
```diff
@@ -90,7 +90,12 @@ const page: IStandardPageMetadata = {
       alt="a satellite in orbit around the earth with a GPS signal coming from it"
       class="w-full max-w-[400px] rounded-2xl"
     />
-    <p>text</p>
+    <p>
+      We're proud to offer surveyors centimeter-level precision measurements through our TURN GPS Network. The TURN
+      Network is a high-precision global navigation satellite system (GNSS) providing real-time corrections and data for
+      post processing. Surveyors in Utah, Nevada, Idaho, and Wyoming are taking advantage of this useful service for a
+      variety of purposes including research, engineering, drone operations, and more!
+    </p>
```
Comment (steveoh) [inline src/pages/solutions.astro:100]: There are more things to offer surveyors than TURN 

image

**Review by steveoh — CHANGES_REQUESTED** [review]
A lot of this needs to be reworked to incorporate the items we decided to add from the [lucid](https://lucid.app/lucidspark/df1452f4-4a93-4073-ae2c-649473e11776/edit?invitationId=inv_0cef03fc-8ede-45a7-9880-0d399ad75cd8&page=0_0#). We offer much more to surveyors than TURN and the solutions sections should market those and entice/educate surveyors to use those things. This is a lot of duplication of the TURN product page.

File: src/pages/solutions/for-surveyors.astro (hunk 1 above)
Comment (seanfernandez) [reply src/pages/solutions/for-surveyors.astro:53]: Understood, please publish what has been provided then I'll start looking closer on how to modify it.

**Pull request closed without merging**